python/test/t_IntegrationExpansion_std.py

```diff
@@ -0,0 +1,149 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import math as m
+
```

Comment:
Please use the `ishigami_function` use case. An example of this is provided in the PR for Sobol' indices in the unit test [t_FunctionalChaosSobolIndices_std.py](https://github.com/mbaudin47/openturns/blob/32cdf26a4eb6076ad9db59a72530f1ea4bc2d137/python/test/t_FunctionalChaosSobolIndices_std.py#L11).
```suggestion
from openturns.usecases import ishigami_function
# Problem parameters
```
This will greatly shorten the code and make it more robust. @sofianehaddad has a similar comment below.

Reply:
I understand that you will not update the unit test, because you are short of time until the release. This can be done afterwards, because it is mainly a maintenance issue. This is why I suggest to resolve the conversation, so that we can focus on what matters.